app/assets/stylesheets/partials/header_footer.scss

```diff
@@ -0,0 +1,25 @@
+/*both*/
+  .img_item {
```

Comment:
der Name lässt meinen, dass es sich immer um ein Image in einem Item geht, aber oftmals hat das Item mit der Klasse kein Image

Reply:
auch würd ich das File wahrscheinlich zwischen header & footer trennen